swap_meet/electronics.py

```diff
@@ -1,2 +1,9 @@
-class Electronics:
-    pass
+from .item import Item
+
+class Electronics(Item):
+
+    def __init__(self, condition=0, age=0):
+        super().__init__(condition=condition, age=age, category="Electronics")
```

Comment:
Nice choice to make sure no one can pass a different category to the children classes. We always want a Clothing object to have the category "Clothing", so there's no reason to even give the option 😄